cloudnatix/README.md

```diff
@@ -0,0 +1,50 @@
+# Agent Check: CloudNatix
+
+## Overview
+
+This check provides data from [CloudNatix][1].
+
+CloudNatix connects to multiple VM & K8s Clusters, enabling automated capacity, cost & operation optimization using patent-pending Autopilot technology. CloudNatix Insights also gives DevOps teams proactive visibility into potential capacity and availability issues, turbocharging productivity.
+
+The CloudNatix integration provides cost and operational optimization insights to DataDog with an out-of-the-box dashboard, allowing you to quickly view cluster cost and analyze opportunities to reduce cost.
```

Comment:
```suggestion
The CloudNatix integration provides cost and operational optimization insights to Datadog with an out-of-the-box dashboard, allowing you to quickly view cluster cost and analyze opportunities to reduce cost.
```

Reply:
Looks like this is still capitalized here

Reply:
updated.